pkg/store/bucket.go

```diff
@@ -189,9 +189,8 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 	m.blockLoadDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
 		Name:    "thanos_bucket_store_block_load_duration_seconds",
 		Help:    "The total time taken to load a block in seconds.",
-		Buckets: []float64{0.1, 0.5, 1, 10, 20, 30, 60, 120},
+		Buckets: []float64{0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 15, 30, 60, 90, 120, 300},
```

Comment:
I am thinking about maybe we can start from a larger number like 1s instead of having multiple small buckets from 0.01 to 0.5. 
WDYT? I feel like we don't care much about loading blocks fast. For example, we load block from disk cache.

Reply:
Same might apply to other two metrics you added

Reply:
I agree. At least for the `thanos_bucket_store_block_load_duration_seconds` and the `indexheader_download_duration_seconds`, it may not be reasonable to expect it to take less than 100ms.

But the `indexheader_load_duration_seconds` should take much less right? I'll keep the smaller buckets on that metric.